src/components/StandardForm.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div>
+    <div class="form-group row" v-for="col in cols" :key="col.field">
+      <label :for="col.field" class="col-sm-2 col-form-label">{{ col.label }}</label>
+
+      <div class="col-sm-10" >
+        <select
+          v-if="col.selectItems && col.selectItems.length > 0"
+          class="form-control"
+          :id="col.field"
+          v-model="record[col.field]"
+          :readonly="col.readonly"
+          :class="classesForField(col.field)"
+          :aria-describedby="`feedback-${col.field}`"
+          >
+          <option v-for="opt in col.selectItems" :key="opt.value" :value="opt.value">
+            {{ opt.label }}
+          </option>
+        </select>
+
+        <input
+          v-else
+          type="text"
```

Comment:
I don't feel that strongly, but I'd prefer to add a `type` attribute to each col.

Especially, if we ever add support for more than select and text inputs, I think it would help the code read more clearly.  Then we can switch explicitly on that type value when rendering inputs.

Reply:
i don't have a good source of the type data. in the follow-up pr, i add an `isDate: true` which is how `RecordForm` did it. so far, we have strings, dates, and selects.